docs/part-7.md

```diff
@@ -0,0 +1,267 @@
+---
+id: part-7
+title: Part 7. Functional Programming Introduction
+---
+
+# Functional Programming
+
+Contributors:
+
+- Sergi Dote
+
+## Introduction
+
+Functional programming is a paradigm that puts the attention on functions. But this definition can be a little confusing since developers are coding using functions and methods and this doesn't mean they are programming following the FP paradigm.
+
+FP talks about abstract control flows, data mapping, operations, immutability, and other concepts we are reviewing in the next steps.
```

Comment:
suggestion: Use *Functional Programming* everywhere instead of *FP* to avoid abbreviations for a topic that the reader is just learning.
**Correct all instances if accepting this suggestion.**
```suggestion
Functional Programming talks about abstract control flows, data mapping, operations, immutability, and other concepts we are reviewing in the next steps.
```

Reply:
Solved every single occurrence!